packages/cubejs-schema-compiler/src/compiler/transpilers/ImportExportTranspiler.ts

```diff
@@ -50,10 +50,9 @@ export class ImportExportTranspiler implements TranspilerInterface {
           }
         });
         const addExportCall = t.callExpression(t.identifier('addExport'), [t.objectExpression(<t.ObjectProperty[]>declarations)]);
-        if (path.get('declaration')) {
+        if ('declaration' in path.node && path.node.declaration) {
           path.replaceWithMultiple([
-            // @todo fix without any
-            (<any>path.get('declaration')).node,
+            path.node.declaration,
```

Comment:
We need any kind of test for this one, just to protect a regression for the feature. @KSDaemon could we add a small tiny test for it?

Huge thanks!

Reply:
@ovr Done!